scripts/ci-integration.sh

```diff
@@ -57,12 +59,19 @@ install_kustomize() {
 
 build_containers() {
    VERSION="$(git describe --exact-match 2> /dev/null || git describe --match="$(git rev-parse --short=8 HEAD)" --always --dirty --abbrev=8)"
+   export CONTROLLER_GEN="${PWD}/${TOOLS_BIN_DIR}/controller-gen"
+   export CONVERSION_GEN="${PWD}/${TOOLS_BIN_DIR}/conversion-gen"
+   export KUSTOMIZE="${PWD}/${TOOLS_BIN_DIR}/kustomize"
+   export GOBINDATA="${PWD}/${TOOLS_BIN_DIR}/go-bindata"
+   export GOBINDATA_CLUSTERCTL_DIR="cmd/clusterctl/config"
+   export CERTMANAGER_COMPONENTS_GENERATED_FILE="cert-manager.yaml"
```

Comment:
I don't think we need these defined here, they seem to be replicating the defaults that already exist in the Makefile

Reply:
@detiber Sorry I thought maybe it is required to declare these variables locally to the file.

Reply:
my fault for letting it get past my review, i'll file an issue to look into this

Reply:
No worries, @prankul88 has already filed a followup PR.